core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/popup/info/UIViewMetadataForm.java

```diff
@@ -173,6 +177,19 @@ public void execute(Event<UIViewMetadataForm> event) throws Exception {
               }
               if(!node.hasProperty(name) || (node.hasProperty(name) && !node.getProperty(name).getString().equals(value)))
                 node.setProperty(name, value);
+            } else if (requiredType == 4){ // double
+              UIFormInput uiInput = uiForm.getUIInput(inputName);
+              double value = 0;
+              if(uiForm.getUIInput(inputName) != null) {

```

Comment:
Plz use variable uiInput instead of "uiForm.getUIInput(inputName)" and you should also test on emptiness of value of the field ( StringUtils.isNotBlank(uiInput.getValue()) )